--- ccontavalli-mod-xslt-337e290/sapi/apache2/Makefile.in~	2011-12-16 17:31:52.000000000 +0100
+++ ccontavalli-mod-xslt-337e290/sapi/apache2/Makefile.in	2012-01-30 16:19:21.146563014 +0100
@@ -96,7 +96,7 @@
 install-stamp: mod_xslt.la Makefile
 	test -w '$(DESTDIR)$(APXS_INSTALLDIR)' || \
 	  { echo 'You are not allowed to write in $(DESTDIR)$(APXS_INSTALLDIR). Are you root??'; exit 1; }
-	$(INSTALL) $(STRIP) -m 0644 -d '$(DESTDIR)$(APXS_INSTALLDIR)'
+	$(INSTALL) $(STRIP) -m 0755 -d '$(DESTDIR)$(APXS_INSTALLDIR)'
 	$(LIBTOOL) --mode=install $(INSTALL) $(STRIP) -m 0644 mod_xslt.la '$(DESTDIR)$(APXS_INSTALLDIR)'
 	@echo Removing extra files...
 	FILE=`grep dlname mod_xslt.la | sed -e "s/[^']*'\([^']*\).*/\1/"` && \
